frontend-react/.env.development

```diff
@@ -1,6 +1,17 @@
+; this is set by env var being set `CLIENT_ENV: development` in the github action (or in the package.json run command)
 REACT_APP_TITLE=CDC Prime ReportStream DEV
-REACT_APP_BACKEND_URL=http://localhost:7071/api/
+REACT_APP_DESCRIPTION=ReportStream Dashboard Application
+REACT_APP_ICON=https://reportstream.cdc.gov/assets/cdc-logo.svg
+
 REACT_APP_BASE_URL=http://localhost:3000
+REACT_APP_BACKEND_URL=http://localhost:7071/api/
```

Comment:
If we change the use in `AuthResource` to use `REACT_APP_BACKEND_URL` we'll want to make this `localhost:3000` so the proxy picks it up and we don't have CORS errors when running RS outside of Docker on M1s

Reply:
Oh, so this is covered by the `"proxy": "http://localhost:7071",` in the package.json, right?